src/pages/styledPages/Profile.styled.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { Box } from "@mui/material";
+
+export const StyledProfileWrapper = styled(Box)`
+  max-width: 1440px;
```

Comment:
i also mentioned this, each and every page should have the same wrapper in which width is defined

Reply:
I create component pageWrapper now and it will do the job